assets/scss/components/_quick-search.scss

```diff
@@ -4,19 +4,25 @@
 
 $quick-search-icon: './../icons/search.svg';
 
-.c-quick-search .c-form__input {
-    padding: 1rem 1rem 1rem 3rem;
-    min-width: 21.375rem;
+.c-quick-search {
+    display: flex;
+    justify-content: center;
+    align-items: center;
 
-    color: $color-neutrals-500;
-    font-family: $font-regular;
-    font-size: 1rem;
-    font-style: normal;
-    font-weight: 400;
-    line-height: 1.5;
+    .c-form__input {
```

Comment:
Please don't nest selectors and don't mix them. If form needs to be styled differently for this particular case, just create modifier class `c-form__input--search`